files/tasks/checkout_packit.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- set_fact:
+    reverse_dir: "{{ project_dir }}/rev_project_cloned"
+
+- name: Install ansible
+  dnf:
+    name:
+    - ansible
+    state: present
+  become: true
+- name: Install flexmock
+  pip:
+    name: flexmock
+  become: true
+- name: Checkout packit
+  git:
+    clone: yes
+    repo: https://github.com/lachmanfrantisek/packit.git
```

Comment:
Testing with this change in packit: https://github.com/packit-service/packit/pull/564

(Will be removed.)

Reply:
Removed.